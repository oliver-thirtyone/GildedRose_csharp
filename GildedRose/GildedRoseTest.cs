using System.Collections.Generic;
using ApprovalTests;
using ApprovalTests.Reporters;
using NUnit.Framework;

namespace GildedRose
{
    [TestFixture]
    [UseReporter(typeof(DiffReporter))]
    public class GildedRoseTest
    {
        [Test]
        public void ConjuredItemQuality_DecreasesCorrectly()
        {
            var items = new List<Item>
            {
                new Item
                {
                    Name = "Conjured",
                    Quality = 42,
                    SellIn = 30
                }
            };

            var app = new GildedRose(items);
            app.UpdateQuality();
            
            Approvals.Verify(items[0]);
        }
    }
}