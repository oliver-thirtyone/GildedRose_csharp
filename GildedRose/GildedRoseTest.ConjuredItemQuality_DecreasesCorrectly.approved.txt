Conjured, 29, 40